pkg/addons/default/assets/coredns-1.29.json

```diff
@@ -171,7 +171,7 @@
                   "-conf",
                   "/etc/coredns/Corefile"
                 ],
-                "image": "%s.dkr.ecr.%s.%s/eks/coredns:v1.11.1-eksbuild.4",
+                "image": "%s.dkr.ecr.%s.%s/eks/coredns:v1.11.1-eksbuild.preview",
```

Comment:
Mmmmm not sure if this is correct^

Reply:
According to https://docs.aws.amazon.com/eks/latest/userguide/managing-coredns.html - this should be `v1.11.1-eksbuild.6`